sdk/monitor/azquery/custom_client.go

```diff
@@ -27,28 +28,49 @@ type LogsClientOptions struct {
 	azcore.ClientOptions
 }
 
+// LogsClient contains the methods for the LogsClient group.
+// Don't use this type directly, use NewLogsClient() instead.
+type LogsClient struct {
+	host string
+	pl   runtime.Pipeline
+}
+
+// MetricsClient contains the methods for the Metrics group.
+// Don't use this type directly, use NewMetricsClient() instead.
+type MetricsClient struct {
+	host string
+	pl   runtime.Pipeline
+}
+
 // NewLogsClient creates a client that accesses Azure Monitor logs data.
 func NewLogsClient(credential azcore.TokenCredential, options *LogsClientOptions) *LogsClient {
 	if options == nil {
 		options = &LogsClientOptions{}
 	}
-	authPolicy := runtime.NewBearerTokenPolicy(credential, []string{"https://api.loganalytics.io/.default"}, nil)
+
+	host := cloud.AzurePublic.Services[monitorQueryLogs].Endpoint
+	if c, ok := options.Cloud.Services[monitorQueryLogs]; ok {
+		host = c.Endpoint
+	}
+	authPolicy := runtime.NewBearerTokenPolicy(credential, []string{host + "/.default"}, nil)
 	pl := runtime.NewPipeline(moduleName, version, runtime.PipelineOptions{PerRetry: []policy.Policy{authPolicy}}, &options.ClientOptions)
-	return &LogsClient{pl: pl}
+	return &LogsClient{host: host + "/v1", pl: pl}
```

Comment:
Should the `/v1` go in the cloud config?

Reply:
Host is also used in line 55 to create the scope, and the scope doesn't use "/v1". It's easier and cleaner to add "/v1" in line 57 then having to strip the string of it in line 55.

Reply:
I missed that, makes sense.

Reply:
Updated in commit [3772df1](https://github.com/Azure/azure-sdk-for-go/pull/19462/commits/3772df1fc545f3ba93ce4eb8c7ba45c582aff39d) to set "/v1" in cloud config